src/stream/src/executor/eowc/eowc_gap_fill.rs

```
@@ -94,7 +97,7 @@ impl<S: StateStore> ExecutorInner<S> {
                     Ok(timestamp) => timestamp,
                     Err(_) => {
                         warn!("Failed to convert timestamptz to timestamp: {:?}", ts);
-                        return Ok(filled_rows);
+                        return Ok(());
```

It should return an error.